config/default_records/request_types/020_limber_high_throughput_bge.wip.yml

```diff
@@ -0,0 +1,37 @@
+---
+# Y24-366
+limber_bge_pcr_free:
+  asset_type: Well
+  order: 1
+  request_class_name: IlluminaHtp::Requests::StdLibraryRequest
+  billable: true
+  product_line_name: Illumina-HTP
+  name: Limber BGE PCR Free
+  acceptable_purposes:
+    - BGE Cherrypick
+  library_types:
+    - BGE
+# For moving PCR Free plates to ISC plates
+# e.g., BGE Lib XP2 -> BGE LIB PCR
+limber_bge_transition:
+  asset_type: Well
+  order: 1
+  billable: true
+  product_line_name: Illumina-HTP
+  name: Limber BGE Transition
+  request_class_name: IlluminaHtp::Requests::StdLibraryRequest
+  acceptable_purposes:
+    - BGE Lib XP2
+  library_types:
+    - BGE
+limber_bge_isc:
+  asset_type: Well
+  order: 1
+  billable: true
+  product_line_name: Illumina-HTP
+  name: Limber BGE ISC
+  request_class_name: Pulldown::Requests::IscLibraryRequest
+  acceptable_purposes:
+    - BGE Lib PCR XP
+  library_types:
+    - BGE
```

Comment:
Is library type 'BGE' for all 3 of these despite them being different library preps? Seems wrong if they want to analyse the sequencing results differently. Maybe check with users.

Reply:
BGE Library type is already across the pipelines (and submissions) that area already in used. It is essentially all part of one bigger pipeline where all the samples are recombined and sequenced together.